fastfield_codecs/src/column.rs

```diff
@@ -240,6 +240,7 @@ where
     // and we do not have any specialized implementation anyway.
 }
 
+/// Wraps anything that returns an iterator into a `Column`.
```

Comment:
```suggestion
/// Wraps an iterator into a `Column`.
```

Reply:
!? What is the deal with min_value and max_value?

Reply:
Indeed that's a bug, it's only used in tests though